src/Common/src/Interop/Windows/user32/Interop.GetWindowText.cs

```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32)]
+        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextW")]
```

Comment:
Today we only run the UAPAOT tests on x64 which is why I didn't see if this was a problem, but if it is then we can revert my change and add some #ifdefs around the EntryPoint so that it only sets it for uapaot builds, or we could have two files that get includded depending on the archGroup